src/scripts.js

```diff
@@ -1 +1,50 @@
-console.log("Hello World");
\ No newline at end of file
+const cardName = document.querySelector(".card-name");
+const address = document.querySelector(".address");
+const email = document.querySelector(".email");
+const dailyStepGoal = document.querySelector(".daily-step-goal");
+const averageStepGoal = document.querySelector(".average-step-goal")
+const welMessage = document.querySelector(".wel-message")
+
+let trialData1 = { 
+  "id": 2,
+  "name": "Christopher Jonson",
+  "address": "123 5th Street, Denver CO 12345",
+  "email": "Christopher@hotmail.com",
+  "strideLength": 4,
+  "dailyStepGoal": 20000,
+  "friends": [ 1, 3, 4] 
+};
+
+let trialData2 = { 
+  "id": 10,
+  "name": "Mar Matlak",
+  "address": "321 100th Street, Boulder CO 54321",
+  "email": "Mar@hotmail.com",
+  "strideLength": 10,
+  "dailyStepGoal": 10000,
+  "friends": [ 10, 30, 40]
+};
+
+let user1 = new User(trialData1)
+let user2 = new User(trialData2)
+let userRepo = new UserRepository([user1, user2])
+
+let loadCardInfo = (user, userRepo) => {
```

Comment:
Nice use of arguments to work with both of our classes, I am curious how we will define our displayed "user" in the future. We may be able to have a refactor that only accepts UserRepository and we use index to access the specific user. (Thats well down the road though)